src/olympia/devhub/tests/test_tasks.py

```diff
@@ -536,7 +536,15 @@ def test_run_linter_path_doesnt_exist(self):
 
         assert exc.value.message == (
             'Path "doesntexist" is not a file or directory or '
-            'does not exist.\n')
+            'does not exist.')
+
+    def test_run_linter_large_output(self):
+        result = json.loads(
+            tasks.run_addons_linter(get_addon_file('bookmark.xpi')))
```

Comment:
`FILE_UPLOAD_MAX_MEMORY_SIZE` is 25 megs by default, so it would not change anything for that file. IMHO you could just do a test that proves the path with the temporary file works by using `override_settings` to set a lower value in the test, and you wouldn't need the big bookmark.xpi file (which is only 4.something megs and would not trigger the temporary file path currently)